File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
    int left = 0,right = s.length-1;

    while(left<right){
        char str = s[left];
        s[left] = s[right];
        s[right] = str;
        left++;
        right--;
    }

    }
}

File: 0344-reverse-string/0344-reverse-string.c
void reverseString(char* s, int sSize) {
    int left = 0,right = sSize-1;

    while(left < right){
        char tem = s[left];
        s[left] = s[right];
        s[right] = tem;
        left++;
        right--;
    }
    
}